### ISSUE TITLE
WCAG 2.2 本体 : 用語「アクセシビリティ サポーテッド (accessibility supported)」の注記 4

### ISSUE THREAD

**Issue by caztcha** [waic/wcag22#2174]
原文
> When citing web content technologies that have multiple versions, the version(s) supported should be specified.

現行訳
> 複数のバージョンが存在するウェブコンテンツ技術を引用する場合は、サポートされているバージョンを明記するのがよい。

specified を「明記する」→「特定する」に修正する。

修正案
> 複数のバージョンが存在するウェブコンテンツ技術を引用するときは、サポートされているバージョンを特定するとよい。
